configs/recognition/tsm/README.md

```diff
@@ -62,7 +62,7 @@ We use efficient setting as default provided in config files, and it can be chan
 ```python
 ...
 # `test_cfg = dict(average_clips=None)` for efficient setting
-test_cfg = dict(average_clips='prob', test_crops=3, twice_sample=True)  # for accurate setting
+test_cfg = dict(average_clips='prob')  # for accurate setting
```

Comment:
It is for efficient setting. And acutally, the `average_clips='prob'` is the same with `average_clips=None`, since when the `crop=1` there is no need to ensemble.

Reply:
Ok, I got it, the modified code will automatically calculate the `test_crops`